ophyd/signal.py

```diff
@@ -453,9 +454,11 @@ def derived_from(self):
 
     def describe(self):
         '''Description based on the original signal description'''
-        desc = self._derived_from.describe()[self._derived_from.name]
+        desc = super().describe()[self.name]  # Description of this signal
         desc['derived_from'] = self._derived_from.name
-        return {self.name: desc}
+        # Description of the derived signal 
+        derived_desc = self._derived_from.describe()[self._derived_from.name]
+        return {self.name: dict(ChainMap(desc, derived_desc))}
```

Comment:
do we need to flatten the chainmap?  If so, might as well do `derived_desc.update(desc)`